### DISCUSSION TITLE
Странности в поведении zapret

### DISCUSSION THREAD

**Discussion by Yatore0** [bol-van/zapret#973] [Q&A]
Последние 3 недели заметил странности в поведении запрета. В моменте у меня может отвалиться дс или ютуб, либо все вместе. Решаеться путем переподбора стратегии, но меня смущает частота происходящего(может произойти раз в пару дней, а может и пару раз за день).
Затригерить гарантированно можно путем перезапуска системы. Есть-ли какой-либо вариант как избежать подобного, либо хотя бы уменьшить частоту.
Я новичек в деле настройки подобного софта, надеюсь на помощь и понимание. Заранее спасибо всем.

**Comment by electrifying** [comment]
если стратегия нестабильно работает, ищите ту что будет работать нормально
